task/data/linked_claim.graphql

```diff
@@ -1,20 +1,23 @@
-type LinkedClaim @createModel( 
```

Comment:
Lets change this model after all to be IEClaim

There are a bunch of changes in here I don't want in the generic LinkedClaim model

Reply:
We are really trying to align to a generic claim model.  The only two fields that you called out were 

1. "claimed_by", which is a wallet address.  We need that somewhere in the claim model and perhaps you have that as a different field and we didn't recognize it.  
2. root_claim_id - This field is being used to get at your Approved and Earned requirement.  Bacalhau compute needs a way to link the earned Claims with the Approved claim.  
Let me know if it's easier to do a quick convo on this instead....I'd like to get us to a common model.